scripts/freefood.coffee

```diff
@@ -0,0 +1,52 @@
+# Description:
+#   Announce free food.
+#
+# Commands:
+#   <botname> free <food> <location>
+
+module.exports = (robot) ->
+  robot.respond /free (.+)/i, (msg) ->
```

Comment:
Yeah I initially was trying to use regex to get match groups/captures but couldn't find clear documentation so I gave up and just messed around with the string :sob: I'll give it another go.